microsoft-edge/extensions-chromium/developer-guide/minimize-page-load-time-impact.md

```diff
@@ -0,0 +1,251 @@
+
+
+<!-- ------------------------------ -->
+#### Profile your content script using Microsoft Edge DevTools
+
+DevTools provides a set of features for inspecting, debugging, and profiling the code that a webpage uses. DevTools can also be used to profile your extension's code.
+
+In this section, you'll learn how to use the **Performance** tool in DevTools to profile your extension's content script. To learn more about the **Performance** tool, see [Introduction to the Performance tool](../../devtools-guide-chromium/evaluate-performance/index.md).
+
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
+
+1. In DevTools, on the **Activity Bar**, select the **Performance** (![Performance tool icon](./minimize-page-load-time-impact-images/performance-tool-icon.png)) tab.  If that tab isn't visible, select **More tools** (![More tools icon](./minimize-page-load-time-impact-images/more-tools-icon.png)) > **Performance**.
+
+1. To start recording a performance profile, click **Record** (![Record icon](./minimize-page-load-time-impact-images/record-icon.png)).
+
+1. Reload the page to capture the profiling data that corresponds to the page load time and then, once the page has finished loading, click **Stop** (![Stop icon](./minimize-page-load-time-impact-images/stop-icon.png)) to end the recording. DevTools displays the recorded performance profile.
+
+   ![The recorded performance profile displayed in the DevTools Performance tool](./minimize-page-load-time-impact-images/profile.png)
+
+1. To search for the performance events that are caused by your content script, press **Ctrl+F** on Windows/Linux or **Command+F** on macOS.  The **Find** text box appears at the bottom of the **Performance** tool.
+
+1. Type **Evaluate script** and then press **Enter** until the **Performance** tool highlights the performance events that are caused by your content script.  You will know that you have found the right performance event when the **Script**  label in the **Summary** panel shows the name of your content script.
+
+   ![The Evaluate script performance event due to the extension's content script running during the page load](./minimize-page-load-time-impact-images/evaluate-script-event.png)
+
+
+<!-- ------------------------------ -->
+#### Profile your content script using the Edge tracing tool
+
+The Edge tracing tool, available at the `edge://tracing` URL, is a powerful tool that can provide detailed analysis of your extension's performance. In this section, you'll learn how to use the Edge tracing tool to understand the impact of your extension on the page load time. To learn more about the Edge tracing tool, which is based on the **Perfetto** tool, see [Perfetto UI](https://perfetto.dev/docs/visualization/perfetto-ui).
```

Comment:
w/o seeing rendered, ambig whether:
* the tool to trace edges
* Microsoft Edge tool for tracing